src/components/Explorer/GlomerulusSchematic.js

```
@@ -18,27 +18,48 @@ function HubMapGlomSchema({
+                case "Efferent_Arteriole_Endothelial_Cell":
+                    console.log("Clicked on Efferent Arteriole Endothelial Cell");
+                    setActiveCell(CellTypeEnum.EFFERENT_ARTERIOLE_ENDOTHELIAL_CELL);
+                    setActiveTab('4');
+                    break;
+                case "Juxtaglomerular_Granular_Cell":
+                    console.log("Clicked on Juxtaglomerular_Granular_Cell");
+                    setActiveCell(CellTypeEnum.JUXTAGLOMERULAR_GRANULAR_CELL);
+                    setActiveTab('4');
+                    break;
+                default:
+                    let ontologyId = event.detail.representation_of;
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)